tests/test_main_loop.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from scapy.all import rdpcap
+from pathlib import Path
+from src.dashboard.alerts.alert import Alert, AlertType, Severity
+from scapy.layers.inet import Ether, IP
+from src import run_config
+from unittest.mock import patch
+
+from src.database.models import DeviceInformation
+from src.signature_detection.signature_detector import SignatureDetector
+from src.signature_detection.ip_signature import IPSignature
+from src.signature_detection.mac_address_signature import MACAddressSignature
+
+
+class TestMainLoop(unittest.TestCase):
+    def setUp(self) -> None:
+        packets = rdpcap(str(Path("test_data/test_tcpdump2.pcap").resolve()))
+        self.src_malicious_ip_packet = packets[0]
+        self.dst_malicious_ip_packet = packets[1]
+        self.src_malicious_ip_packet[IP].src = "62.210.205.141"
+        self.dst_malicious_ip_packet[IP].dst = "5.188.62.140"
+        self.ids_signatures = [IPSignature("192.168.1.0/24"), MACAddressSignature()]
+        self.signature_detector = SignatureDetector(self.ids_signatures)
+
+    # TODO: fill this test in
```

Comment:
Remove or update

Reply:
Removed since the test is now not necessary